panther_driver/src/battery_node.py

```diff
@@ -165,13 +146,13 @@ def _publish_battery_msg(self, bat_pub, present, V_bat=NaN, temp_bat=NaN, I_bat=
             battery_msg.present = True
         else:
             battery_msg.header.stamp = rospy.Time.now()
-            battery_msg.voltage = NaN
-            battery_msg.temperature = NaN
-            battery_msg.current = NaN
-            battery_msg.percentage = NaN
-            battery_msg.capacity = NaN
-            battery_msg.design_capacity = NaN
-            battery_msg.charge = NaN
+            battery_msg.voltage = float('nan')
+            battery_msg.temperature = float('nan')
+            battery_msg.current = float('nan')
+            battery_msg.percentage = float('nan')
+            battery_msg.capacity = float('nan')
+            battery_msg.design_capacity = float('nan')
+            battery_msg.charge = float('nan')
             battery_msg.power_supply_status
             battery_msg.power_supply_health
             battery_msg.power_supply_technology = 3
```

Comment:
Here we can use BatteryState msg constant to be more descriptive

Reply:
Use
``` python
BatteryState.POWER_SUPPLY_TECHNOLOGY_LIPO
```